### DISCUSSION TITLE
Just wanted to say thank you!

### DISCUSSION THREAD

**Discussion by themadwriter** [basujindal/stable-diffusion#171] [General]
I have a GTX 1650 4GB and so far this has been the only fork which has allowed me to run SD. I´m forever grateful! Not even the new webgui lowvram option that took its idea from this repo worked on my laptop. Without this repo I would have never been able to play with this new technology and experience it. 

Thank you very much and I look forward to whatever improvements are made in the future 🥇

**Comment by basujindal** [comment]
Thank you for the appreciation, I have a 6Gb GPU, and when SD was first released, I was also frustrated about not being able to run such a wonderful model locally.
